iota/commands/extended/prepare_transfer.py

```diff
@@ -130,6 +133,7 @@ def __init__(self):
 
         # Optional parameters.
         'changeAddress': Trytes(result_type=Address),
+        'securityLevel': f.Choice([1, 2, 3]) | f.Optional(default=AddressGenerator.DEFAULT_SECURITY_LEVEL),
```

Comment:
❔ Could we make this match the filters for the other commands?  This will ensure that the error codes/messages for invalid inputs remain consistent.

Reply:
ok, I'll make it with Min and Max. Thought that `security_level` semantically is more an enumeration than range of integers :)